XPNet.GraphicsTest/XPNet.GraphicsTest.csproj.user

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Ditto here on the .user file.  Notice how this file has a local path to where things are on _your_ computer in it.  That wouldn't work for anyone else.  To resolve, delete from git and add to the .gitignore instead.

Reply:
Done